google-cloud-firestore/src/main/java/com/google/cloud/firestore/UserDataConverter.java

```diff
@@ -160,7 +160,8 @@ static Value encodeValue(
       Map<String, Object> map = (Map<String, Object>) sanitizedObject;
 
       for (Map.Entry<String, Object> entry : map.entrySet()) {
-        Value encodedValue = encodeValue(path.append(entry.getKey()), entry.getValue(), options);
+        Value encodedValue =
+            encodeValue(path.append(entry.getKey(), false), entry.getValue(), options);
```

Comment:
```suggestion
            encodeValue(path.append(entry.getKey(), /* splitPath= */ false), entry.getValue(), options);
```

I generally prefer these inline comments for non-descriptive values.

Reply:
Done